sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/http/HttpResponse.java

```diff
@@ -68,6 +69,13 @@
      */
     public abstract Flux<ByteBuf> body();
 
+    /**
+     * Get the response content as InputStream.
+     *
+     * @return this response content as InputStream
+     */
+    public abstract Flux<InputStream> bodyAsInputStream();
```

Comment:
why can't we just rely on Flux<Byte> or Flux<String> ?

we are not doing attachment anymore, so we shouldn't need to do InputStream I think

Reply:
Yes, attachment is no longer supported, will get rid of this completely.